### PR TITLE
Updates to add person card and news

### PR DESCRIPTION
**Pull request by rowanc1**
image

image

image

image

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -6,18 +6,17 @@ title: Home
...
 
 ## IDEA Statement
 
 Everyone deserves to work a safe, inclusive and equitable environment. As educators and researchers, we are committed to fostering such an atmosphere in academia. We actively mentor students from underrepresented groups, implement fair hiring practices, and promote global scientific collaboration. We strive to create inclusive spaces, prevent discrimination, hold ourselves and others accountable, and continuously educate ourselves on equity issues. By embracing diverse perspectives and backgrounds, we can advance scientific progress and build a more equitable academic community that reflects and serves our diverse society.
 
-
-
-
-
+::::{template:list} news.yml
```
Comment (rowanc1) [inline index.md:17]: This is in the newest release of curvenote CLI, to be released shortly!

File: research.md [hunk 2]
```diff
@@ -2,15 +2,15 @@
...
 ---
 
-
-
-## 4DSTEM 
+## 4DSTEM
 
 ### Experiments
 
++++ {"class": "col-"}
```
Comment (rowanc1) [inline research.md:9]: This is a hack to make float work. We will do better in the near future.

File: people.md [hunk 3]
```diff
@@ -2,87 +2,88 @@
...
-:alt: Colin
-:class: float-right
-:width: 200px
-:align: right
-```
+:::{person}
+:name: Colin Ophus
+:position: Group Leader
+:pronouns: he/him
+:orcid: 0000-0003-2348-8558
```
Comment (rowanc1) [inline people.md:9]: The person card will be released shortly.

File: people.md [hunk 4]
```diff
@@ -2,87 +2,88 @@
...
-:alt: Steph
-:class: bg-primary
-:width: 200px
-:align: right
-```
-
+:::{person}
+:name: Stephanie Ribet
+:position: Postdoctoral Researcher
+:pronouns: she/her
```
Comment (rowanc1) [inline people.md:31]: I have added a space for pronouns, obviously up to you if you want to use that and also add to the rest of the people on this page.

**Review by rowanc1 — COMMENTED** [review]
I will be releasing the Curvenote CLI shortly which has a few built in plugins (person cards and templated lists.)

File: index.md (hunk 1 above)
Comment (rowanc1) [reply index.md:19]: This has been released!

File: news.yml [hunk 5]
```diff
@@ -0,0 +1,153 @@
+news:
+  - title: Launch of Elemental Microscopy covered in Nature
+    url: https://www.nature.com/articles/d41586-024-02577-1
+    date: August 2024
+    image: https://media.nature.com/lw767/magazine-assets/d41586-024-02577-1/d41586-024-02577-1_27432946.jpg
+  - title: New Technique Achieves Unprecedented View of Complex Nanostructure
```
Comment (rowanc1) [inline news.yml:6]: The news, minus the first item were the items on the foundry website.

**Comment by cophus** [comment]
Very cool!

**Pull request merged**